ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -58,5 +59,6 @@ if [ -f "${OPERATOR_DIR}/tests/scorecard/config.yaml" ]; then
     --output json \
     --wait-time 3000s \
     --service-account "${SCORECARD_SERVICE_ACCOUNT}" \
-    "${OPERATOR_DIR}" > "${ARTIFACT_DIR}"/scorecard-output-custom.json || true
+    "${OPERATOR_DIR}" > "${ARTIFACT_DIR}"/scorecard-output-custom.json || true \
+    --verbose
```

Comment:
@jsztuka you need to pass `--verbose` to the `operator-sdk` and not to the `/bin/true`

Reply:
modified